swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory else []
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+    def get_by_id(self, search_id):
+        if isinstance(search_id, int):
+            for i in self.inventory:
+                if i.id != search_id:
+                    continue
+                else:
+                    return i
+            return None
+        else:
+            return "Invalid input. Id must be a number."
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        else:
+            self.remove(my_item)
+            self.add(their_item)
+            other_vendor.remove(their_item)
+            other_vendor.add(my_item)
+        return True
```

Comment:
Again, aim to have the least amount of code nested as possible. You don't necessarily need the `else` block here. 
```suggestion
        if my_item not in self.inventory or their_item not in other_vendor.inventory:
            return False

        self.remove(my_item)
        self.add(their_item)
        other_vendor.remove(their_item)
        other_vendor.add(my_item)
        
        return True
```
@kathryn-bello and @Bipentium25